src/omt/integer_optimizer.cpp

```diff
@@ -74,17 +71,15 @@ OptimizationResult OMTOptimizerInteger::optimize(
   return OptimizationResult(OptimizationResult::OPTIMAL, value);
 }
 
-OptimizationResult OMTOptimizerInteger::minimize(
-    SmtEngine* parentSMTSolver, TNode target)
+OptimizationResult OMTOptimizerInteger::minimize(SmtEngine* optChecker,
+                                                 TNode target)
 {
-  return this->optimize(
-      parentSMTSolver, target, OptimizationObjective::MINIMIZE);
+  return this->optimize(optChecker, target, true);
 }
-OptimizationResult OMTOptimizerInteger::maximize(
-    SmtEngine* parentSMTSolver, TNode target)
+OptimizationResult OMTOptimizerInteger::maximize(SmtEngine* optChecker,
+                                                 TNode target)
 {
-  return this->optimize(
-      parentSMTSolver, target, OptimizationObjective::MAXIMIZE);
+  return this->optimize(optChecker, target, false);
```

Comment:
In case you haven't, please make sure this (and the rest of the changes) are clang-formatted.

Reply:
didn't I? will do now

Reply:
they are formatted